lib/delighted/http_response.rb

```diff
@@ -37,5 +42,17 @@ def get_header_value(key)
         end
       end
     end
+
+    def parse_link_header(header_value)
+      links = {}
+      # Parse each part into a named link
+      header_value.split(',').each do |part, index|
+        section = part.split(';')
+        url = section[0][/<(.*)>/,1]
+        name = section[1][/rel="(.*)"/,1].to_sym
```

Comment:
I think the [spec](https://tools.ietf.org/html/rfc8288#section-3) allows bare `rel=next` as well as `rel="next"`. It's probably not a important since I think in practice we will always have quoted values here.